suite/tests/CMakeLists.txt

```diff
@@ -3011,8 +3011,18 @@ if (CLIENT_INTERFACE)
                 client-interface/memval-test.c "" "" "")
           endif(X86 AND (NOT X64) AND UNIX)
         elseif (sample STREQUAL "opcode_count")
-          torunonly_ci(sample.${sample} common.${control_flags} ${sample}
-            common/${control_flags}.c "-opcode 5" "" "")
+          # We do not do a simple sanity test for opcode_count, but a normal test
+          # that checks the sample's output.
+          # FIXME i#4372: The test is currently only supported on X86 32-bit.
+          # The test program is implemented in pure asm (for UNIX).
```

Comment:
So should we do the sanity test on other platforms then?

Reply:
Done.